openapi/openapi.json

```diff
@@ -735,14 +735,14 @@
           "Info"
         ],
         "summary": "Get Info",
-        "operationId": "get_info_optimade_v0.10.0_info_get",
+        "operationId": "get_info_optimade_v0_10_0_info_get",
```

Comment:
Where does this change from `v0.10.0` to `v0_10_0` come from?

Reply:
So I'm confused about this too; it seems to depend on whether you do an editable install when running pre-commit! I changed the gh workflow to use an editable install and now the checks pass, but I don't understand why.

Reply:
> Have you tried to test if the package data is included with the new setup?

Yep, it Works For Me™, I think adding a CI test for non-editable install is probably overkill...

Reply:
> > Have you tried to test if the package data is included with the new setup?
> 
> Yep, it Works For Me™, I think adding a CI test for non-editable install is probably overkill...

Great. That's all I wanted to check. No need for CI for sure.

Reply:
> So I'm confused about this too; it seems to depend on whether you do an editable install when running pre-commit! I changed the gh workflow to use an editable install and now the checks pass, but I don't understand why.

Very interesting!
Well --- good catch in any case. Glad to see this finally passing.